Misc/NEWS.d/next/Library/2018-01-25-21-04-11.bpo-27931.e4r52t.rst

```diff
@@ -0,0 +1 @@
+Email parse IndexError <""@wiarcom.com> patch made by xiang.zhang
```

Comment:
I suggest: Fix email address header parsing error when the username is an empty quoted string.  Patch by xian.zhang.